frontend/src/components/contracts/ContractDetails.tsx

```diff
@@ -0,0 +1,127 @@
+import moment from "moment";
+
+import React from "react";
+
+import { Row, Col } from "react-bootstrap";
+
+import ContractsApi from "../../libraries/explorer-wamp/contracts";
+
+import CardCell from "../utils/CardCell";
+import TransactionLink from "../utils/TransactionLink";
+
+interface Props {
+  accountId: string;
+}
+
+interface State {
+  locked?: boolean;
+  transactionHash?: string;
+  timestamp?: number;
+}
+
+export default class extends React.Component<Props, State> {
+  state: State = {
+    locked: true
+  };
+
+  getContractInfo = async () => {
```

Comment:
Non-descriptive function name: this function does not get a contract info to the caller.